src/renderers/dom/shared/dangerousStyleValue.js

```diff
@@ -48,7 +51,27 @@ function dangerousStyleValue(name, value) {
   }
 
   if (typeof value === 'string') {
```

Comment:
I don't think this check is enough; we should not fire the warning if the string already contains the 'px' at the end.